src/grafana/influxdb/docker/docker-compose.yml

```diff
@@ -1,24 +1,18 @@
 version: '3.3'
 services:
 
-  # Prometheus uses the image resulting form the build
-  # of `./influxdb` which simply packs some configuration
-  # into the form of an image.
-  #
-  # It could instead, use volume-mounts.
+  # Prometheus uses the InfluxDB datasource
```

Comment:
would it be `Grafana uses the InfluxDB datasource`?

Reply:
+1